scripts/index.js

```diff
@@ -1 +1,153 @@
-console.log(`There are ${characters.length} characters in the array.`);
\ No newline at end of file
+console.log(`There are ${characters.length} characters in the array.`);
+
+//For each of these questions, write a function that returns the answer.
+
+//How many characters names start with "A"?
+
+function char_a(characters) {
+    let char_a = [];
+    characters.forEach(function (character) {
+        if (character.name[0] === "A" && character != "") {
+        char_a.push(character.name)
+        };
+    })
+    console.log(char_a)
+}
+
+// Output 168 + list of names
+//===============================================//
+
+//How many characters names start with "Z"?
+function char_z(characters) {
+    let char_z = [];
+    characters.forEach(function (character) {
+        if (character.name[0] === "Z" && character != "") {
+        char_z.push(character.name)
+        };
+    })
+    console.log(char_z)
+}
+// output 8 + list of names
+//===============================================//
+
+
+//How many characters are dead?
+function char_dead(characters) {
+    let char_dead = [];
+    characters.forEach(function (character) {
+        if (character.died !== "") {
+        char_dead.push(character.name)
+        };
+    })
+    console.log(char_dead)
+}
+
+// output 553 + list of names
+//===========================================//
+
+
+//Who has the most titles?
+function title_length(characters) {
```

Comment:
Lol so there is much better way to solve this problem than what I'm about to share with. Truthfully, I got held up on it for a few hours so I feel my solution needs to be refactored. Maybe we can review it and see if we come up with a better idea to solve it without using two loops. 

`function title_length(characters) {
    let temp = 0;
    
    for (let character in characters) {
        let currentLength = characters[character].titles.length;
        if (currentLength > temp) {
            temp = currentLength;
        };
        
    }
    for (let character in characters) {
        let currentLength = characters[character].titles.length;
        if (temp === currentLength) {
            console.log(currentLength)
            let charName = characters[character].name;
            return charName;  
        }
    }
}
title_length(characters)`
Otherwise, great job pushing through...It was a toughy.